lib/matplotlib/dates.py

```diff
@@ -1184,11 +1193,20 @@ def set_data_interval(self, vmin, vmax):
         self._wrapped_locator.set_data_interval(vmin, vmax)
         return DateLocator.set_data_interval(self, vmin, vmax)
 
-    def __call__(self, *args, **kwargs):
-        vmin, vmax = self.axis.get_view_interval()
-        vmin *= MUSECONDS_PER_DAY
-        vmax *= MUSECONDS_PER_DAY
-        ticks = self._wrapped_locator.tick_values(vmin, vmax)
+    def __call__(self):
```

Comment:
Can you add a note to https://github.com/matplotlib/matplotlib/tree/master/doc/api/api_changes about this?  It is a harmless change (as the inputs were ignored), but it should still be documented.